client.go

```diff
@@ -89,6 +93,11 @@ func Dial(addr string, opts ...ConnOption) (*Client, error) {
 	return &Client{conn: c}, err
 }
 
+// Accept accepts an AMQP server connection.
+func Accept(l net.Listener, opts ...ConnOption) (*Client, error) {
```

Comment:
Feeling we could just let the user handle doing the a tcp or tls Listener and accept the net.Conn.  Then the user uses the ConnServer() /w the existing New func. That would keep the API a bit cleaner I think.

Reply:
Thats how it works now, check the latest.